lib/index.ts

```diff
@@ -59,12 +59,84 @@ export type ProgressCallback = (eventObj: any) => void;
 
 const DEFAULT_PROGRESS_BAR_STEP_COUNT = 50;
 
+// https://github.com/moby/moby/blob/v26.1.5/api/types/types.go#L597
+// https://github.com/moby/moby/blob/v26.1.5/distribution/push_v2.go#L256
+// https://github.com/moby/moby/blob/v26.1.5/plugin/backend_linux.go#L462
+interface ApiTypes_PushResult {
+	Tag: string;
+	Digest: string;
+	Size: number;
+}
+
+// https://github.com/opencontainers/image-spec/blob/v1.1.1/specs-go/v1/descriptor.go#L22
+interface ImageSpecV1_ImageSpecDecriptor {
```

Comment:
```suggestion
interface ImageSpecV1_ImageSpecDescriptor {
```

Reply:
Nice spot. I will actually just reduce it to `ImageSpecV1_Descriptor`. It seems that I added the prefix and didn't simplify the suffix part to match the Go type linked above as I was intending...